src/kernel/src/rtld/module.rs

```diff
@@ -318,7 +318,7 @@ impl<E: ExecutionEngine> Module<E> {
 
     pub fn dump<P: AsRef<Path>>(&mut self, path: P) -> Result<(), std::io::Error> {
         let path = path.as_ref();
-        let mut file = File::create(path)?;
+        let mut file = OpenOptions::new().write(true).create_new(true).open(path)?;
```

Comment:
This one should always overwrite the destination file otherwise the dump will be an old one from the previous run.

Reply:
Alrighty.